aspnetcore/migration/Logging-nonaspnetcore.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Migrate from Microsoft.Extensions.Logging 2.1 to 2.2 or 3.0
+author: pakrym
+description: This article outlines the common steps for migrating a non-ASP.NET Core application that uses Microsoft.Extensions.Logging 2.1 to Microsoft.Extensions.Logging 2.2 or 3.0.
+ms.author: pakrym
+ms.custom: mvc
+ms.date: 01/04/2019
+uid: migration/logging-nonaspnetcore
+---
+
+# Migrate from Microsoft.Extensions.Logging 2.1 to 2.2 or 3.0
+
+This article outlines the common steps for migrating a non-ASP.NET Core application that uses `Microsoft.Extensions.Logging` 2.1 to `Microsoft.Extensions.Logging` 2.2 or 3.0.
+
+## 2.1 to 2.2
+
+Manually create `ServiceCollection` and call `AddLogging`.
```

Comment:
He'll correct me if I'm wrong, but @scottaddie and I have been linking all of the first mentions of API to the API Browser. Still not possible yet for 2.2 AFAIK ... I think the API Browser is still broken.